docs/how-to-guides/access-ceph-dashboard.md

```diff
@@ -0,0 +1,41 @@
+# Accessing Ceph dashboard
+
+## Contents
+
+* [Introduction](#introduction)
+* [Presumption](#presumption)
```

Comment:
Why call it `Presumption` ? Lets stay consistent with `Prerequisites` as in this context having `rook` and `rook-ceph` component deployed is a prerequisite.